### PR TITLE
Hide sensitive info url

### LINKED ISSUE #31: Hide tokens and authorization info in the generated report
## Description

Add the possibility to hide tokens and authorization info in the generated report to avoid expose sensitive information via [configuration file](https://github.com/scanapi/scanapi/blob/124983ca0365e4552f7ad56a6f9523a829c6c293/scanapi/settings.py#L5) (usually `.scanapi.yaml`).

Change the sensitive information value to `<sensitive_information>`

Configuration Options:
- `report`
-- `hide-response` or `hide-request`
--- `headers` or `body` or `url`
---- list of keys to hide

Example:
```yaml
report:
  hide-response:
    headers:
      - Authorization
      - api-key
  hide-response:
    body:
      - api-key
```

The logic is implemented inside the [hide_sensitive_info method](https://github.com/scanapi/scanapi/blob/124983ca0365e4552f7ad56a6f9523a829c6c293/scanapi/utils.py#L25)

Example of how this should be rendered in the reports:

image

- [x] header for request
- [x] header for response
- [x] body for request
- [x] body for response
- [ ] url for request
- [ ] url for response

### PR DESCRIPTION
**Pull request by gillianomenezes**
## Description

Add the possibility to hide tokens and authorization info in the generated report to avoid expose sensitive information via [configuration file](https://github.com/scanapi/scanapi/blob/124983ca0365e4552f7ad56a6f9523a829c6c293/scanapi/settings.py#L5) (usually `.scanapi.yaml`).

Change the sensitive information value to `SENSITIVE_INFORMATION`

Configuration Options:
- `report`
-- `hide-response` or `hide-request`
--- `headers` or `body` or `url`
---- list of keys to hide

Example:
```yaml
report:
  hide-response:
    headers:
      - Authorization
      - api-key
    url:
      - secret-key
  hide-response:
    body:
      - api-key
```

So the following example
```
URL: http://test.com/users/34afaaff-628c-4dbf-b93b-7ef4595eb092
```
Would be rendered like this in the reports:
```
URL: http://test.com/users/SENSITIVE_INFORMATION
```
This PR closes #31

### CODE REVIEW

File: tests/unit/test_utils.py [hunk 1]
```diff
@@ -165,6 +167,21 @@ def test_overrides(self, response):
...
         assert response.headers["abc"] == "SENSITIVE_INFORMATION"
 
+    def test_overrides_sensitive_info_url(self, response):
+        secret_key = "129e8cb2-d19c-51ad-9921-cea329bed7fa"
+        response.url = (
+            f"http://test.com/users/129e8cb2-d19c-51ad-9921-cea329bed7fa/details"
+        )
+        http_attr = "url"
+        secret_field = secret_key
+        import pdb
```
Comment (camilamaia) [inline tests/unit/test_utils.py:177]: 😅

File: scanapi/utils.py [hunk 2]
```diff
@@ -1,7 +1,12 @@
 from scanapi.errors import InvalidKeyError, MissingMandatoryKeyError
 from scanapi.settings import settings
 
-ALLOWED_ATTRS_TO_HIDE = ("headers body").split()
+HEADERS = "headers"
```
Comment (camilamaia) [inline scanapi/utils.py:4]: @gillianomenezes what do you think of moving all this hide related functions to a separated file dedicated for that?

File: tests/unit/test_utils.py (hunk 1 above)
Comment (gillianomenezes) [reply tests/unit/test_utils.py:177]: Ops! 😂

File: scanapi/utils.py (hunk 2 above)
Comment (gillianomenezes) [reply scanapi/utils.py:4]: Good idea. I'll do that!

File: tests/unit/test_utils.py [hunk 3]
```diff
@@ -3,12 +3,10 @@
 
 from scanapi.errors import InvalidKeyError, MissingMandatoryKeyError
 from scanapi.utils import (
```
Comment (camilamaia) [inline tests/unit/test_utils.py:5]: I would suggest we could split the test file too, the same way you did for the utils and hide_utils. What do you think?

File: scanapi/hide_utils.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+from scanapi.settings import settings
```
Comment (camilamaia) [inline scanapi/hide_utils.py:1]: 👏

File: tests/unit/test_utils.py (hunk 3 above)
Comment (gillianomenezes) [reply tests/unit/test_utils.py:5]: Good idea! I'll do it

**Pull request merged**